server/neptune/gateway/event/check_run_handler_test.go

```diff
@@ -226,7 +226,7 @@ func TestCheckRunHandler(t *testing.T) {
 			Name:       "atlantis/invalid: testroot",
 		}
 		err := subject.Handle(context.Background(), e)
-		assert.ErrorContains(t, err, "unable to determine root name")
```

Comment:
We should definitely fail if we can't determine the root name.  the name of this test is wrong though, it should say something along the lines of "other atlantis check". Can you change the name?